.changeset/calm-ducks-bake.md

```diff
@@ -0,0 +1,14 @@
+---
+"@osdk/create-app.template.tutorial-todo-aip-app.beta": minor
+"@osdk/create-app.template.tutorial-todo-app.beta": minor
+"@osdk/create-app.template.tutorial-todo-aip-app": minor
+"@osdk/create-app.template.tutorial-todo-app": minor
+"@osdk/create-app.template.react.beta": minor
+"@osdk/create-app.template.vue.v2": minor
+"@osdk/create-app.template.react": minor
+"@osdk/create-app.template.vue": minor
+"@osdk/example-generator": minor
+"@osdk/create-app": minor
```

Comment:
Depending what you need to do with this change (check with team) it may need to be a patch if it has to be cherry picked